tools/tf2openapi/types/tfsignaturedef.go

```diff
@@ -71,6 +72,60 @@ func NewTFMethod(key string, method string) (TFMethod, error) {
 	return tfMethod, nil
 }
 
-func (t *TFSignatureDef) Schema() *openapi3.Schema {
-	return &openapi3.Schema{}
+func (t *TFSignatureDef) Schema() (*openapi3.Schema, error) {
+	if t.Method != Predict {
+		return &openapi3.Schema{}, errors.New("schemas for classify/regress APIs currently not supported")
```

Comment:
Can you make this error message a constant and test it directly in your test. Same with all other validation steps.